e2e/packages/BUILD.bazel

```diff
@@ -35,10 +35,9 @@ nodejs_test(
     data = [
         "npm_determinism.spec.js",
         "@e2e_packages_npm_install//:node_modules/jsesc/package.json",
+        "@e2e_packages_npm_install_duplicate_for_determinism_testing//:node_modules",
     ],
     entry_point = ":npm_determinism.spec.js",
-    node_modules = "@e2e_packages_npm_install_duplicate_for_determinism_testing//:node_modules",
-    # TODO(gregmagolan): fix this test on windows
     tags = ["fix-windows"],
```

Comment:
no more TODO to fix-windows?

Reply:
whoops. didn't mean to remove that thanks.